src/editor/operations.ts

```diff
@@ -181,12 +182,15 @@ export function formatRangeAsCode(range: Range): void {
 
     const hasBlockFormatting = (range.length > 0)
         && range.text.startsWith("```")
-        && range.text.endsWith("```");
+        && range.text.endsWith("```")
+        && range.text.includes('\n');
+    const hasBacktick = (range.text.includes("`"))
+        && !range.text.startsWith("`")
+        && !range.text.endsWith("`");
```

Comment:
I think this variable name should be improved

Reply:
i followed same naming approach as the code above it which says, `hasBlockFormatting`, if you have a better name, kindly suggest one.

Reply:
Yes but this also checks that it doesn't start & end with backticks, which `hasBacktick` doesn't really imply at all